src/types/interfaces.ts

```diff
@@ -58,6 +58,11 @@ export enum FilterOptionsKeywords {
   TEAM = 'ius',
 }
 
+export enum handleSelectorComponentModes {
+  CreateTaskFieldUpdate,
```

Comment:
In typescript enums are generally written in PascalCase since enums are not a core-JS contruct and are more similar to types